packages/calcite-components/src/components/block/block.scss

```diff
@@ -194,7 +198,6 @@ calcite-action-menu {
   }
 }
 
-:host([heading]),
 :host([description]) {
```

Comment:
[description] would also be an issue here because it isn't reflected either. Should this css be removed?